### DISCUSSION TITLE
Setting zoom / default_time_value for a specific CPU graph via Config?

### DISCUSSION THREAD

**Discussion by leeola** [ClementTsang/bottom#519] [Q&A]
### Question

I'm using the layout:

```
[[row]]
  ratio=30
  [[row.child]]
  type="cpu"
[[row]]
    ratio=40
    [[row.child]]
      ratio=4
      type="proc"
      default=true
    [[row.child]]
      ratio=3
      [[row.child.child]]
        type="temp"
      [[row.child.child]]
        type="mem"
[[row]]
  ratio=30
  [[row.child]]
  type="cpu"
```
And i'd like to have the first CPU graph show a longer time span than the second, by default with the config. Currently i manually adjust this every time i open the application. Am i missing how this can be configured? I've not found this in the document, so far.

**Comment by ClementTsang** [comment]
Sorry for the late response - this isn't currently supported, though I could look into it.
